tests/test_ag3.py

```diff
@@ -1066,6 +1072,16 @@ def test_cnv_discordant_read_calls(sample_sets, contig):
     if contig == "X":
         assert ds.dims["variants"] == 29
 
+    dim_dict = {"2R": 40, "3R": 29, "X": 29}
```

Comment:
Opportunity to move this higher up and use it as single point where expectation of number of variants is defined. E.g.:

```
    expected_variants = {"2R": 40, "3R": 29, "X": 29}
    if isinstance(contig, str):
        n_variants = expected_variants[contig]
    elif isinstance(contig, (list, tuple)):
        n_variants = sum([expected_variants[c] for c in contig])

    assert ds.dims["variants"] == n_variants
```